sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -695,6 +719,9 @@ bool test_main(void) {
   } else if (rst_info == kDifRstmgrResetInfoWatchdog) {
     collect_alert_dump_and_compare(kRound2);
 
+    // Do not remove this log statement, as it is used to synchronize with the
+    // DV environment
+    LOG_INFO("Disable ping timer alert check");
```

Comment:
@weicaiyang i think we might need to define another macro with a more obvious name..something like `DV_SYNC_INFO` to make sure these statements don't get removed by others. 

Functionally it should behave the same, but just have a different name to illustrate its purpose.

Reply:
sounds good. added this in the agenda of the next DV meeting